<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphApplicationClass</class>
 <widget class="QMainWindow" name="GraphApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Aplicatie Grafuri</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="detailsZone" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Proiect realizat de</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="orientedRadioButton">
         <property name="text">
          <string>Orientat</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Lista keybind-uri:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Aplicație Grafuri</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>PgUp/PgDown -&gt; Selecteaza toate nodurile cresc/desc</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Tipul grafului:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Enter - Uneste nodurile selectate</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="unorientedRadioButton">
         <property name="text">
          <string>Neorientat</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Del -&gt; Sterge nodurile selectate</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Home -&gt; Deselecteaza nodurile</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>End -&gt; Termina arcele selectate</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Țocu Marius Daniel</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>orientedRadioButton</zorder>
      <zorder>label</zorder>
      <zorder>label_3</zorder>
      <zorder>label_2</zorder>
      <zorder>label_4</zorder>
      <zorder>label_5</zorder>
      <zorder>label_6</zorder>
      <zorder>unorientedRadioButton</zorder>
      <zorder>label_7</zorder>
      <zorder>label_8</zorder>
      <zorder>label_9</zorder>
      <zorder>label_10</zorder>
     </widget>
    </item>
    <item>
     <widget class="DrawingZone" name="drawingZone" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFisier">
    <property name="geometry">
     <rect>
      <x>282</x>
      <y>198</y>
      <width>153</width>
      <height>72</height>
     </rect>
    </property>
    <property name="title">
     <string>Fisier</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionMatrice_adiacenta"/>
   </widget>
   <addaction name="menuFisier"/>
  </widget>
  <action name="actionMatrice_adiacenta">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>Matrice adiacenta</string>
   </property>
   <property name="iconText">
    <string>Matrice adiacenta</string>
   </property>
   <property name="toolTip">
    <string>Deschide fisierul graph.txt daca exista</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawingZone</class>
   <extends>QWidget</extends>
   <header>DrawingZone.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="GraphApplication.qrc"/>
 </resources>
 <connections/>
</ui>
